Source/FixedBackground/FixedBGDensities.hpp

```diff
@@ -0,0 +1,91 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef FIXEDBGDENSITIES_HPP_
+#define FIXEDBGDENSITIES_HPP_
```

Comment:
As before I think it would be better to rename and separate by conserved quantity rather than densities vs fluxes